### PR TITLE
Zkapp tx limit banner

### PR DESCRIPTION
**Pull request by qwadratic**
replace old zkapp banner to another one saying about zkapp tx limits per block

### CODE REVIEW

File: docs/zkapps/o1js/keccak.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ keywords:
 
 :::info
 
-Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/o1js/keccak.mdx:17]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/o1js/merkle-tree.mdx [hunk 2]
```diff
@@ -13,7 +13,7 @@ keywords:
 
 :::info
 
-Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/o1js/merkle-tree.mdx:16]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/o1js/on-chain-values.mdx [hunk 3]
```diff
@@ -15,7 +15,7 @@ keywords:
 
 :::info
 
-Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/o1js/on-chain-values.mdx:18]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/o1js/permissions.mdx [hunk 4]
```diff
@@ -17,7 +17,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/o1js/permissions.mdx:20]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/o1js/recursion.mdx [hunk 5]
```diff
@@ -16,7 +16,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/o1js/recursion.mdx:19]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/o1js/smart-contracts.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/o1js/smart-contracts.md:23]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/o1js/time-locked-accounts.mdx [hunk 7]
```diff
@@ -14,7 +14,7 @@ keywords:
 
 :::info
 
-Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/o1js/time-locked-accounts.mdx:17]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/roadmap.mdx [hunk 8]
```diff
@@ -18,7 +18,7 @@ import Subhead from '@site/src/components/common/Subhead';
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/roadmap.mdx:21]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/01-hello-world.mdx [hunk 9]
```diff
@@ -20,7 +20,7 @@ import ResponsiveVideo from '@site/src/components/common/ResponsiveVideo';
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/01-hello-world.mdx:23]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx [hunk 10]
```diff
@@ -20,7 +20,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx:23]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/03-deploying-to-a-network.mdx [hunk 11]
```diff
@@ -20,7 +20,7 @@ import ResponsiveVideo from '@site/src/components/common/ResponsiveVideo';
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/03-deploying-to-a-network.mdx:23]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/04-zkapp-ui-with-react.mdx [hunk 12]
```diff
@@ -20,7 +20,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/04-zkapp-ui-with-react.mdx:23]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/05-common-types-and-functions.mdx [hunk 13]
```diff
@@ -24,7 +24,7 @@ import ResponsiveVideo from '@site/src/components/common/ResponsiveVideo';
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/05-common-types-and-functions.mdx:27]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/06-offchain-storage.mdx [hunk 14]
```diff
@@ -18,7 +18,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/06-offchain-storage.mdx:21]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/07-oracle.mdx [hunk 15]
```diff
@@ -18,7 +18,7 @@ import ResponsiveVideo from '@site/src/components/common/ResponsiveVideo';
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/07-oracle.mdx:21]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/08-custom-tokens.mdx [hunk 16]
```diff
@@ -17,7 +17,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/08-custom-tokens.mdx:20]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/09-recursion.mdx [hunk 17]
```diff
@@ -18,7 +18,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/09-recursion.mdx:21]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/10-account-updates.mdx [hunk 18]
```diff
@@ -19,7 +19,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/10-account-updates.mdx:22]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/11-advanced-account-updates.mdx [hunk 19]
```diff
@@ -6,8 +6,7 @@ sidebar_label: 'Tutorial 11: Advanced Account Updates'
 
 :::info
 
-Please note that zkApp programmability is not yet available on Mina Mainnet, but
-zkApps can now be deployed to Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/11-advanced-account-updates.mdx:9]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/anonymous-message-board.mdx [hunk 20]
```diff
@@ -6,7 +6,7 @@ sidebar_label: 'Example: Anonymous Message Board'
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/anonymous-message-board.mdx:9]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/tutorials/interacting-with-zkapps-server-side.mdx [hunk 21]
```diff
@@ -6,7 +6,7 @@ sidebar_label: 'Interacting with zkApps Server-Side'
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/interacting-with-zkapps-server-side.mdx:9]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

File: docs/zkapps/zkapps-for-ethereum-developers.mdx [hunk 22]
```diff
@@ -17,7 +17,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted following the mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/zkapps-for-ethereum-developers.mdx:20]:
```suggestion
The maximum number of zkApp transactions per block is currently capped at 24. This restriction will gradually be lifted after the Mainnet upgrade.
```

**Review by barriebyron — COMMENTED** [review]
In most cases, guidance for timeless documentation is applied and I recommend not to use "currently" 
In this ephemeral instance, let's do it so devs know what the limit is

Question for @mitschabaude and @garwalsh Do we want to add an FAQ or other docs about zkApp transaction limits?

**Comment by mitschabaude** [comment]
> replace old zkapp banner to another one saying about zkapp tx limits per block

@qwadratic is this really that bad of a limitation to put a banner on _every_ docs page about it? how about making it a banner on _one_ page?

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 23]
```diff
@@ -3,12 +3,6 @@ title: How to Use a zkApp
 hide_title: true
 ---
 
-:::info
```
Comment (barriebyron) [inline docs/using-mina/how-to-use-zkapp.mdx:6]: The purpose of the callout has changed, so we can omit all of these banners except for the two pages as indicated

File: docs/zkapps/how-to-test-a-zkapp.mdx [hunk 24]
```diff
@@ -11,26 +11,13 @@ keywords:
...
 
-A Docker image for Mina local networks provides a simple and efficient way to deploy and run lightweight Mina blockchain networks for testing zkApps. The `o1labs/mina-local-network` Docker image provides:
-
-- Genesis ledger with pre-funded accounts
-- Accounts manager service
-- Additional port served by reverse proxy to pass requests to the Mina daemon GraphQL endpoint so zkApps work without additional configuration
-- Single commands to launch a single node or multi node network
-
-Pull the [o1labs/mina-local-network](https://hub.docker.com/r/o1labs/mina-local-network) image from Docker Hub.
+A Docker image for Mina local networks provides a simple and efficient way to deploy and run a lightweight Mina blockchain network for testing zkApps. See [Testing zkApps with Lightnet](/zkapps/testing-zkapps-lightnet).
```
Comment (barriebyron) [inline docs/zkapps/how-to-test-a-zkapp.mdx:20]: snuck in an unrelated change here!

File: docs/zkapps/o1js/interact-with-mina.md [hunk 25]
```diff
@@ -21,7 +21,7 @@ keywords:
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will be gradually lifted after the Mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/o1js/interact-with-mina.md:24]: @garwalsh  did we want to add language about the intention behind this limit?

File: docs/zkapps/tutorials/03-deploying-to-a-network.mdx [hunk 26]
```diff
@@ -20,7 +20,7 @@ import ResponsiveVideo from '@site/src/components/common/ResponsiveVideo';
 
 :::info
 
-zkApp programmability is not yet available on the Mina Mainnet. You can get started now by deploying zkApps to the Berkeley Testnet.
+The maximum number of zkApp transactions per block is currently capped at 24. This restriction will be gradually lifted after the Mainnet upgrade.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/03-deploying-to-a-network.mdx:23]: @garwalsh did we want to add language about the intention behind this limit?

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 27]
```diff
@@ -21,11 +15,7 @@ import Subhead from '@site/src/components/common/Subhead';
 1. Install a zkApp-compatible [Mina wallet](../using-mina/install-a-wallet).
 2. Make sure that your Mina wallet contains MINA to pay for transaction fees. 
 
-:::note
-
-The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps currently.
-
-:::
+The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps.
```
[the remark this answers is not in the document]
Comment (barriebyron) [reply docs/using-mina/how-to-use-zkapp.mdx:18]: see https://github.com/o1-labs/docs2/wiki/Docs-Style-Guide#tone-and-content

> Write [timeless documentation](https://developers.google.com/style/timeless-documentation) that avoids words and phrases that anchor the documentation to a point in time or assume knowledge of prior or future products and features. Words to avoid: now, new, soon, not yet, latest, future, and currently.

**Comment by barriebyron** [comment]
@qwadratic Ivan, I will merge this PR if there are no objections, lmk

**Comment by qwadratic** [comment]
@barriebyron no objections, its GTG. I agree its now much better!
